lib/debuglet.js

```diff
@@ -83,42 +84,44 @@ function Debuglet(config, logger) {
  */
 Debuglet.prototype.start = function(opt_callback) {
   var that = this;
-
-  uid.get(that.config_.workingDirectory, function(err, uid) {
-    if (err) {
-      that.logger_.error('Unable to compute unique id of the application', err);
-      if (opt_callback) {
-        opt_callback(err);
-      }
-      return;
-    }
-    that.logger_.info('Unique ID for this Application: ' + uid);
-
-    that.debugletApi_.init(uid, function(err) {
+  scanner.scan(that.config_.workingDirectory, function(err, hash, fileStats) {
```

Comment:
Previous we would do a scan only if `workingDirectory` contained a `package.json`. I think this is important. This prevents us from doing a large scan of the whole file-system accidentally if the node process is started in a weird location. Could you bring back that behaviour?